packer/ansible/agent.yml

```diff
@@ -42,69 +53,31 @@
           - make
           - perl
           - p7zip
+          - https://rpm.nodesource.com/pub_14.x/el/7/{{ ansible_architecture }}/nodejs-{{ node_version }}-1nodesource.{{ ansible_architecture }}.rpm
```

Comment:
I looks we already use NodeJS v16 - https://github.com/Percona-Lab/pmm-submodules/blob/b79a9d61a56c55d4ac6fbd1261bb6837e13d2942/build/rpmbuild-docker/Dockerfile#L4

Reply:
https://github.com/percona-platform/saas-ui/blob/main/package.json#L117